extra/twig-extra-bundle/TwigExtraBundle.php

```diff
@@ -11,8 +11,16 @@
 
 namespace Twig\Extra\TwigExtraBundle;
 
+use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\HttpKernel\Bundle\Bundle;
+use Twig\Extra\TwigExtraBundle\DependencyInjection\Compiler\MissingExtensionSuggestorPass;
 
 class TwigExtraBundle extends Bundle
 {
+    public function build(ContainerBuilder $container)
+    {
+        parent::build($container);
```

Comment:
the parent doesn't have any operation, can it be removed?

Reply:
I think we are doing the same in Symfony, just to be on the safe side if one day we add something in the parent?